lib/Backend/InliningDecider.cpp

```diff
@@ -505,12 +506,14 @@ bool InliningDecider::GetBuiltInInfoCommon(
         *inlineCandidateOpCode = Js::OpCode::InlineArrayPop;
         break;
 
+    case Js::JavascriptBuiltInFunction::JavascriptArray_At:
     case Js::JavascriptBuiltInFunction::JavascriptArray_Concat:
     case Js::JavascriptBuiltInFunction::JavascriptArray_Reverse:
     case Js::JavascriptBuiltInFunction::JavascriptArray_Shift:
     case Js::JavascriptBuiltInFunction::JavascriptArray_Slice:
     case Js::JavascriptBuiltInFunction::JavascriptArray_Splice:
 
+    case Js::JavascriptBuiltInFunction::JavascriptString_At:
```

Comment:
To make this work need to track it through further - would go from here through to inline.cpp also would need helpers in JnHelperMethodList.h.

Alternatively we can do without inliner support - delete these entries AND delete the entries in LibraryFunction.h

Reply:
I better work some more but get the optimization